rollbar/resource_project.go

```diff
@@ -215,6 +250,16 @@ func resourceProjectRead(ctx context.Context, d *schema.ResourceData, m interfac
 func resourceProjectUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	teamIDs := getTeamIDs(d)
 	projectID := mustGetID(d)
+	name := d.Get(projectName).(string)
+
+	timezone := d.Get(projectTimezone).(string)
+	timeFormat := d.Get(projectTimeFormat).(string)
+	if timezone == "" {
```

Comment:
The empty string here is explicit input (user has to set timezone to empty string), or can also mean that the timezone was not specified at all by the user?

Reply:
yeah, since null and "" values are treated the same, setting explicitly timezone = "" would work too.

Reply:
One explanation: there is no really perfect solution. I am assuming that the majority of our users still have default values. If they don't , this is what will happen  to them after this change: 
 ```
 ~ resource "rollbar_project" "project_backend" {
        id            = "665633"
        name          = "backend"
      ~ timezone      = "US/Alaska" -> "US/Pacific"
        # (5 unchanged attributes hidden)
    }
```
it will try to update a non default timezone to a default timezone